test/mock_batch_test.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:cloud_firestore_mocks/cloud_firestore_mocks.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() async {
+  TestWidgetsFlutterBinding.ensureInitialized();
+  String users = "users";
+  MockFirestoreInstance firestore = MockFirestoreInstance();
```

Comment:
In order to execute each test independently, it would be better to move this into each `test(...)`. Would you mind initializing the Fake firestore in each test?

```dart
final firestore = FakeFirebaseFirestore();
```